cmake/Reprojucer.cmake

```diff
@@ -183,6 +184,16 @@ function(jucer_project_settings)
       elseif(tag STREQUAL "PREPROCESSOR_DEFINITIONS")
         string(REPLACE "\n" ";" value "${value}")
 
+      elseif(tag STREQUAL "HEADER_SEARCH_PATHS")
+        string(REPLACE "\\" "/" value "${value}")
+        string(REPLACE "\n" ";" value "${value}")
+        unset(header_search_paths)
```

Comment:
I considered it but decided against, since it matches the same code in `jucer_export_target_configuration`: https://github.com/McMartin/FRUT/blob/3db7295ad0b78dcd1c690eaa65c085fd81d4bc97/cmake/Reprojucer.cmake#L830-L833